src/Microsoft.Experimental.Collections/Microsoft.Experimental.Collections.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <Description>This package provides collections that are being considered for future versions of .NET Core.</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageTags>collections, system, microsoft, fastdictionary, multivaluedictionary</PackageTags>
```

Comment:
As I discovered recently, nuget splits tags on spaces. You should have no commas

Reply:
I don't think anyone will search for "fastdictionary"

Reply:
Ok, will change to spaces.